modules/470-chrony/docs/FAQ_RU.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Модуль chrony: FAQ"
+type:
+  - instruction
+---
+
+## Как запретить использование chrony и использовать ntp-демоны на узлах?
+
+1. Выключите модуль chrony в Deckhouse CM:
+```yaml
+chronyEnabled: "false"
+```
+
+2. Создайте NodeGroupConfiguration custom step чтобы включить ntp-демоны на узлах (пример для systemd-timesyncd):
+```yaml
+apiVersion: deckhouse.io/v1alpha1
+kind: NodeGroupConfiguration
+metadata:
+  name: enable_ntp_on_node.sh
+spec:
+  weight: 100
+  nodeGroups: ["*"]
+  bundles: ["*"]
+  content: |
+    # Copyright 2021 Flant JSC
+    #
+    # Licensed under the Apache License, Version 2.0 (the "License");
+    # you may not use this file except in compliance with the License.
+    # You may obtain a copy of the License at
+    #
+    #     http://www.apache.org/licenses/LICENSE-2.0
+    #
+    # Unless required by applicable law or agreed to in writing, software
+    # distributed under the License is distributed on an "AS IS" BASIS,
+    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    # See the License for the specific language governing permissions and
+    # limitations under the License.
```

Comment:
We don't need license information at all because this is a user's step.

Reply:
But it's our step from our module. It's not a user step.

Reply:
done